OpenRA.Mods.Common/Widgets/Logic/ColorPickerLogic.cs

```diff
@@ -65,6 +73,107 @@ public ColorPickerLogic(Widget widget, ModData modData, World world, HSLColor in
 
 			hueSlider.Value = initialColor.H / 255f;
 			onChange(mixer.Color);
+
+			// Setup tab controls
+			var mixerTab = widget.Get("MIXER_TAB");
+			var paletteTab = widget.Get("PALETTE_TAB");
+			var paletteTabScrollPanel = widget.Get("PALETTE_TAB_SCROLLPANEL");
```

Comment:
This isn't a scrollpanel, so can you please rename to `var palettePanel` and `PALETTE_TAB_PANEL`?